server/utils/utils.ts

```diff
@@ -128,6 +129,14 @@ export const calculateAge = (dateOfBirth: string, dateOfIncident: string) => {
   }
 }
 
+export const getEvidenceCategory = (evidenceArray: EvidenceDetails[], isBaggedAndTagged: boolean) => {
```

Comment:
You could include a return type for this, such as:

`export const getEvidenceCategory = (evidenceArray: EvidenceDetails[], isBaggedAndTagged: boolean): EvidenceDetails[] => {`